tests/test_model_registry.py

```diff
@@ -8,6 +8,8 @@
 
 @pytest.mark.parametrize("model_name", model_registry.keys())
 def test_model_registry(model_name, request):
+    pytest.importorskip("timm")
```

Comment:
Same as above. Perhaps instead we do:
```
if model_name in ['timm']:
    pytest.importorskip("timm")
if model_name in ['unet']:
   pytest.importorskip("monai")
```
We can then skip the try, except below.
```